content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -719,6 +719,65 @@ webhooks:
 
 The `matchPolicy` for an admission webhooks defaults to `Equivalent`.
 
+### Matching requests: matchConditions
+
+{{< feature-state state="alpha" for_k8s_version="v1.27" >}}
+
+{{< note >}}
+Match conditions requires the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+`AdmissionWebhookMatchConditions` to be explicitly enabled before this feature can be used.
+{{< /note >}}
+
+Match conditions can be added to webhooks for additional fine-grained request filtering using
```

Comment:
Maybe mention they can only be used for negative matching after a request was already matched by rules

Reply:
Done.